frontend-ssr/components/exercise-submissions/ExerciseSubmissionsPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { loadSubmissions, searchSubmissions, nextPage } from '../../redux/exercise-submissions/exercise-submissions.actions';
+
+interface Props {
+  dispatch: any;
+}
+function ExerciseSubmissionPage(props: { submissions, dispatch }) {
+  const { submissions, dispatch } = props;
+
+  const onSearchSubmission = (e) => {
+    e.preventDefault();
+    let keyword = e.target.searchKeyword.value;
+    dispatch(searchSubmissions(keyword));
+  };
+
+  function onNextPage() {
+    dispatch(loadSubmissions(submissions.countPerPage));
```

Comment:
No need for 2 separate actions. Let's call just load with the already mocked "new" submissions.